### DISCUSSION TITLE
Studio: show foxglove.geojson points as dots of track

### DISCUSSION THREAD

**Discussion by solderdev** [foxglove/community#1001] [Q&A] [answered]
Hi!

I'd like to visualize an MCAP file containing foxglove.geojson strings like:
`{'geojson': '{"type": "Point", "coordinates": [11.46247838, 42.0578836717]}'}`

Expected result (from Foxglove Studio example):
![Screenshot_20240523_150103](https://github.com/foxglove/community/assets/12023076/7d91e664-2578-485b-9ceb-db95be08deb4)

Actual map view:
![Screenshot_20240523_150454](https://github.com/foxglove/community/assets/12023076/e8e23154-a59e-4db2-90fb-d4aaa6a4354a)

Is it possible to somehow reduce or change the marker size?

Best regards,
Peter

**Comment by defunctzombie** [comment]
Could you provide a sample file with some data so we can look into how it is being displayed.

**Reply by solderdev** [reply]
[NMEA_GNSS.zip](https://github.com/user-attachments/files/15786964/NMEA_GNSS.zip)

**Comment by defunctzombie** [comment, marked as answer]
Thanks for the feedback. The next version of the app will allow you to configure the point display mode for GeoJSON topics to change the "pin" to a "dot".
